code/game/gamemodes/gang/gang.dm

```diff
@@ -69,13 +80,33 @@ GLOBAL_LIST_INIT(gang_colors_pool, list("red","orange","yellow","green","blue","
 	if(gangs.len < 2) //Need at least two gangs
 		return 0
 
-	return 1
+	//turn off sec and captain
+	SSjob.DisableJob(/datum/job/captain)
+	SSjob.DisableJob(/datum/job/hos)
+	SSjob.DisableJob(/datum/job/warden)
+	SSjob.DisableJob(/datum/job/detective)
+	SSjob.DisableJob(/datum/job/officer)
 
+	return 1
 
-/datum/game_mode/gang/post_setup()
+/datum/game_mode/gang/send_intercept()
 	set waitfor = FALSE
-	..()
-	sleep(rand(10,100))
+	..(FALSE, FALSE)
+
+	priority_announce("Incoming Bluespace Artillery Strike", "Target location: Security department. Evacuate immediately", 'sound/machines/Alarm.ogg')
+
+	sleep(100)
+
+	explosion(pick(get_area_turfs(target_armory)), 7, 14, 28, 30, TRUE, TRUE)
+
+	sleep(1200)
+
+	for(var/datum/gang/G in gangs)
+		for(var/datum/mind/boss_mind in G.bosses)
+			to_chat(boss_mind.current, "<span class='danger'>You've decided to take charge of the situation and form a gang. X and Y are your ganghead friends [syndicate company] has offered an uplink to help you take over, it will be delivered in five minutes.</span>")
```

Comment:
[syndicate company] should pick from a list of the factions, or just the name of the gang has offered or something

X and Y should be the names of the other gang heads